### DISCUSSION TITLE
Download only subtitles

### DISCUSSION THREAD

**Discussion by Bmswad1** [JunkFood02/Seal#133] [Q&A] [answered]
I tried to write a command to only download video subs from YouTube in srt format but it didn't work
The command
```
--write-subs en --convert-subs "srt"
```

**Comment by JunkFood02** [comment, marked as answer]
add `--skip-download` to prevent downloading videos. Use regex to download subtitles with suffix `"en.*"`

try `--sub-langs "en.*" --write-subs --convert-subs "srt" --skip-download`
